From 1ede5cc0bcc1a1805970fe941596b33ba5b60136 Mon Sep 17 00:00:00 2001
From: Frank Denis <github@pureftpd.org>
Date: Mon, 10 Oct 2016 14:43:57 +0200
Subject: [PATCH] $MAKE_TOOLCHAIN --force

---
 dist-build/android-build.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dist-build/android-build.sh b/dist-build/android-build.sh
index 82eac53..360677b 100755
--- a/dist-build/android-build.sh
+++ b/dist-build/android-build.sh
@@ -35,7 +35,7 @@ echo
 echo "Building for platform [${NDK_PLATFORM}], retaining compatibility with platform [${NDK_PLATFORM_COMPAT}]"
 echo
 
-bash $MAKE_TOOLCHAIN --platform="$NDK_PLATFORM_COMPAT" \
+bash $MAKE_TOOLCHAIN --force --platform="$NDK_PLATFORM_COMPAT" \
     --arch="$ARCH" --install-dir="$TOOLCHAIN_DIR" || exit 1
 
 ./configure \
@@ -50,7 +50,7 @@ if [ "$NDK_PLATFORM" != "$NDK_PLATFORM_COMPAT" ]; then
   echo
   echo "Configuring again for platform [${NDK_PLATFORM}]"
   echo
-  bash $MAKE_TOOLCHAIN --platform="$NDK_PLATFORM" \
+  bash $MAKE_TOOLCHAIN --force --platform="$NDK_PLATFORM" \
       --arch="$ARCH" --install-dir="$TOOLCHAIN_DIR" || exit 1
 
   ./configure \
-- 
2.9.3 (Apple Git-75)

